contexts/v1.jsonld

```diff
@@ -2,9 +2,9 @@
   "@context": {
     "@protected": true,
 
-    "RevocationList2020Credential": {
+    "StatusList2021Credential": {
       "@id":
-        "https://w3id.org/vc-revocation-list-2020#RevocationList2020Credential",
+        "https://w3id.org/vc/status-list#StatusList2021Credential",
```

Comment:
this link is dead.

https://w3id.org/vc/status-list#StatusList2021Credential

Reply:
The link should be working now.  w3id.org has been updated.